ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/management_interfaces.schema.yml

```
@@ -43,3 +43,6 @@ keys:
         ipv6_gateway:
           type: str
           description: IPv6 address of default gateway in management VRF
+        mac_address:
+          type: str
+          description: MAC_address
```

description is used as the help text for the user so it would look better like this:

```suggestion
          description: MAC address
```

you will need to re-build the schema after this